### PR TITLE
48005 implement partial to trainee user show

### PR DESCRIPTION
**Pull request by mdmahmudurrahman**

### CODE REVIEW

**Comment by manazirahsan** [comment]
ok

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -16,4 +15,10 @@ def link_to_remove_fields label, f
     field + link
   end
 
+  def course_completed_percentage
+    completed_tasks = UserSubject.where(user_id: current_user.id, status:
```
Comment (dieunb) [inline app/helpers/application_helper.rb]: use rails `scope` instead of `where` condition pls

File: app/views/shared/_trainee_progress.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div class="well"><h4><%= t :course_completed_text %>(<%= course_completed_percentage %>%)</h4>
```
Comment (manazirahsan) [inline app/views/shared/_trainee_progress.html.erb]: 80 chars.

**Comment by manazirahsan** [comment]
okey

**Pull request merged**